mlpf/pytorch_delphes/gravnet.py

```diff
@@ -0,0 +1,122 @@
+from typing import Optional, Union
+from torch_geometric.typing import OptTensor, PairTensor, PairOptTensor
+import time
+
+import torch
+from torch import Tensor
+from torch.nn import Linear
+from torch_scatter import scatter
+from torch_geometric.nn.conv import MessagePassing
+
+try:
+    from torch_cluster import knn
+except ImportError:
+    knn = None
+
+# copied it from pytorch_geometric source code
+# ADDED: retrieve edge_index, retrieve edge_weight
+# CHANGED: self.lin -> self.lin_p
+# CHANGED: used reduce='sum' instead of reduce='mean' in the message passing
+# REMOVED: skip connection
+
+class GravNetConv(MessagePassing):
```

Comment:
this seems duplicated with respect to gravnet_LRP.py, can you make a common import?